src/common/catalog/catalog-entity.ts

```diff
@@ -293,11 +293,22 @@ export interface CatalogEntitySettingsMenu {
   };
 }
 
+export interface CatalogEntityContextMenuNavigate {
```

Comment:
I wonder if there are manual changes in this merge. Hard to tell. Would be easy with rebase?

Reply:
No manual changes unless of conflicts. I didn't want to rebase and break your very long review

Reply:
Ah, of course :) Appreciated.